fk/luae_na/src/main/java/se/inera/intyg/common/luae_na/v1/model/converter/RespConstants.java

```diff
@@ -21,6 +21,40 @@
 
 public class RespConstants {
 
+    public static final String DIAGNOS_CATEGORY_ID = "diagnos";
+    public static final String DIAGNOS_SVAR_ID_6 = "6";
+    public static final String DIAGNOS_BESKRIVNING_DELSVAR_ID_6 = "6.1";
+    public static final String DIAGNOS_DELSVAR_ID_6 = "6.2";
+    public static final String BIDIAGNOS_1_BESKRIVNING_DELSVAR_ID_6 = "6.3";
+    public static final String BIDIAGNOS_1_DELSVAR_ID_6 = "6.4";
+    public static final String BIDIAGNOS_2_BESKRIVNING_DELSVAR_ID_6 = "6.5";
+    public static final String BIDIAGNOS_2_DELSVAR_ID_6 = "6.6";
+    public static final String DIAGNOS_SVAR_JSON_ID_6 = "diagnoser";
+    public static final String DIAGNOS_CATEGORY_TEXT_ID = "KAT_3.RBK";
+    public static final String DIAGNOS_SVAR_TEXT_ID = "FRG_6.RBK";
+    public static final String DIAGNOS_SVAR_DESCRIPTION_ID = "FRG_6.HLP";
+    public static final String DIAGNOS_DELSVAR_TEXT = "DFR_6.2.RBK";
```

Comment:
One that slipped through that should have _ID on the end.

Reply:
Fixed!